### ISSUE TITLE
Add missing descriptions to builtin payload classes

### ISSUE THREAD

**Issue by postmodern** [ronin-rb/ronin-payloads#81]
Some of the payload classes in `lib/ronin/payloads/builtin/` are missing `description`s:

* [ ] `lib/ronin/payloads/builtin/cmd/awk/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/bash/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/lua/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/node/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/openssl/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/perl/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/php/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/powershell/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/python/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/cmd/ruby/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/java/reverse_shell.rb`
* [ ] `lib/ronin/payloads/builtin/java/reverse_shell/Payload.java.erb`

Descriptions must be defined like:

```ruby
  description <<~DESC
    Text goes here.
    
    May contain multiple paragraphs.
  DESC
```

**Comment by ervinismu** [comment]
are the `descriptions` in this issue is refer to `class` comments and `method` comments ?

for example 
- description 1 : line 29 - 31
- description 2 : line 38 - 40

<img width="680" alt="Screenshot 2023-05-27 at 14 26 34" src="https://github.com/ronin-rb/ronin-payloads/assets/18729060/d3cbb687-10df-49aa-8d1e-198c3854a1d7">

**Comment by postmodern** [comment]
@ervinismu the `description <<~DESC...` method which adds the description metadata to the class, which can then be printed or displayed by [ronin-app](https://github.com/ronin-rb/ronin-app).

**Comment by postmodern** [comment]
@ervinismu er I now see what you mean. You could re-use the class'es top-level documentation description for the `description <<~DESC` text. That's what I did for the `builtin/shellcode/` payload classes. Ideally, the `description <<~DESC` text should be more verbose, as users will see the text.

**Comment by ervinismu** [comment]
yes @postmodern , i meant the text of description.
based on my screenshot in prev comment, the description in this file will be like this, it is connect ?

```ruby
module Ronin
  module Payloads
    module CMD
      module Awk
        #
        # A basic `awk` reverse shell command.
        #
        class ReverseShell < CommandPayload
          include Mixins::ReverseShell
          register 'cmd/awk/reverse_shell'

          description <<~DESC
              Builds the `awk` reverse shell command.
          DESC

          #
          # Builds the `awk` reverse shell command.
          #
          def build
          end
        end
      end
    end
  end
end
```

and all class under `lib/ronin/payloads/builtin/cmd/` already have description, it is ok to reuse description in every class or need to edit it ?

**Comment by postmodern** [comment]
@ervinismu that looks about right, just indent by 2 spaces instead of 4, otherwise rubocop will flag it. Any `builtin/` payload class without a `description <<~DESC` should have one at a minimum. Maybe later we or someone else can add more verbose multi-sentence descriptions to these payloads.

**Comment by postmodern** [comment]
Closed by PR #86.
